recipe/meta.yaml

```diff
@@ -21,7 +21,8 @@ requirements:
     - python
     - pip
     - cython
-    - numpy
```

Comment:
I'm not sure about these changes. This version bumps the minimum required numpy to 1.12.
Following https://conda-forge.org/docs/meta.html#building-against-numpy, I would want something like

```
host:
  - numpy 1.12.*
run:
  - {{ pin_compatible('numpy') }}
```

But NumPy 1.12 (and 1.13) don't support Python 3.7, so we need numpy 1.14.* for py37.

Reply:
When running locally `conda build --python=3.7 recipe/ `

conda can't resolve the right dependencies

```
conda.exceptions.UnsatisfiableError: The following specifications were found to be in conflict:
  - numpy=1.12 -> python[version='>=2.7,<2.8.0a0']
  - python=3.7
Use "conda info <package>" to see the dependencies for each package.
```